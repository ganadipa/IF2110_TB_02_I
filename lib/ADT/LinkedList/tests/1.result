lib/ADT/LinkedList/tests/1.in lib/ADT/LinkedList/tests/1.out: PASSED!
