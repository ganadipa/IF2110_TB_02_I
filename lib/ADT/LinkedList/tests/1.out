[3,2,102,1002]